Mathlib/Topology/GDelta/UniformSpace.lean

```diff
@@ -0,0 +1,10 @@
+/-
+Copyright (c) 2019 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel, Yury Kudryashov
+-/
+
+import Mathlib.Topology.GDelta.MetrizableSpace
```

Comment:
The imports in the deleted file were
```
import Mathlib.Topology.GDelta.Basic
import Mathlib.Order.Filter.CountableInter
import Mathlib.Topology.UniformSpace.Basic
```
I presume you checked that these new imports imply the previous ones?